### PR TITLE
Fix 307 p3

### PR DESCRIPTION
**Pull request by lynxlynxlynx**
Final part of #307, before deciding on the stragglers (please take a look at the lists).

### CODE REVIEW

File: externalized/game.json [hunk 1]
```diff
@@ -55,6 +55,12 @@
     //  vanilla value: 0
     "enemy_weapon_minimal_status": 0,
 
+    //  Make the AI a bit smarter and let it target heads and legs sometimes
+    //  vanilla values: false, 0, 0
+    "mod_ai_a": false,
```
Comment (selaux) [inline externalized/game.json]: Can we call this differently? Maybe `ai_targets_body_parts`? Or more generic (so we can add stuff later) `mod_ai_aiming`?

File: Build/Tactical/Soldier_Create.cc [hunk 2]
```diff
@@ -1254,6 +1255,8 @@ void CreateDetailedPlacementGivenBasicPlacementInfo( SOLDIERCREATE_STRUCT *pp, B
 			break;
 		case SOLDIER_CLASS_ELITE:
 			pp->bExpLevel = (INT8) 6 + bExpLevelModifier;
+			if(pp->bExpLevel<GCM->getGamePolicy()->enemy_elite_minimum_level) pp->bExpLevel=GCM->getGamePolicy()->enemy_elite_minimum_level;
```
Comment (selaux) [inline Build/Tactical/Soldier_Create.cc]: Isn't there a a comparison missing here?

File: Build/Tactical/Soldier_Create.cc [hunk 3]
```diff
@@ -1336,6 +1339,7 @@ void CreateDetailedPlacementGivenBasicPlacementInfo( SOLDIERCREATE_STRUCT *pp, B
 	pp->bWisdom				= (INT8)(bBaseAttribute + Random( 9 ) + Random( 8 ));
 	pp->bMorale				= (INT8)(bBaseAttribute + Random( 9 ) + Random( 8 ));
 
+	if(ubSoldierClass==SOLDIER_CLASS_ELITE && pp->bExpLevel>GCM->getGamePolicy()->enemy_elite_maximum_level)
```
Comment (selaux) [inline Build/Tactical/Soldier_Create.cc]: Body of the if statement is missing?

File: Build/Tactical/Soldier_Create.cc (hunk 2 above)
Comment (lynxlynxlynx) [inline Build/Tactical/Soldier_Create.cc]: No, it's just bad style. In one of the following commits I cleaned it up (the abstraction one).

File: Build/Tactical/Soldier_Create.cc (hunk 3 above)
Comment (lynxlynxlynx) [inline Build/Tactical/Soldier_Create.cc]: Looking at it, it's more correct to only run the next line if these two conditions are true.

**Comment by lynxlynxlynx** [comment]
Cleaned it up further and reverted a bit.

**Pull request merged**